docs/_docs/versions.md

```diff
@@ -7,112 +7,161 @@ redirect_from: /docs/
 
 # Manage Versions of Your Zapier Integration
 
-Your Zapier integration should be quite nice when first launched—minimal, perhaps, but with the basics covered. The app authentication will let users connect to your API, and the triggers and actions should get data to and from your API.
+Your Zapier integration should be ready to use when first launched—minimal, perhaps, but with the basics covered. The app authentication will let users connect to your API, and the triggers and actions should get data to and from your API.
 
 Over time, though, things will change. Users will request new triggers and actions, or want to send and receive more data with your existing ones. You’ll add new features to your app and want to add them to your Zapier integration, too. And eventually, your API may change, and your Zapier integration’s authentication or API calls may need to be changed as well.
 
 That’s where versions come in. Zapier lets you create multiple versions of your integration to test and build new features without interrupting existing users. You can then release the new version to some users, ensure everything is working, and eventually migrate all of your users to the new version. After that, you can deprecate and remove the old version.
 
-## Manage Integration Versions in Zapier
+This guide will cover the steps for _Cloning_, _Promoting_, _Migrating_, and _Deprecating_ integration versions in more detail. Put briefly, the process of creating a new version of your integration takes three steps in either the Developer Platform [Visual Builder](https://developer.zapier.com/) or [Zapier Command Line Interface](https://platform.zapier.com/docs/export) (CLI).
 
-![Zapier integration versions](https://cdn.zapier.com/storage/photos/4294ed1a2f6c7e3980cb9ac9c43f8655.png)
+In the Visual Builder:
 
-Zapier’s Platform UI includes a _Versions_ page in the left sidebar. New integrations provide a starter `1.0.0` version, and you can create other versions as needed.
+1. In the sidebar on the left, navigate to **Manage**, then **Versions**
+2. Select the gear icon next to the version you’d like to update, then select **Clone**
+3. Select a new version number, then **Clone** and **Edit** to create the new version and begin editing it
 
-Click the gear icon beside a version on the _Versions_ page to make changes. You can _Clone_ your integration to make a new version based on the features of an existing version. Once the new version is ready, you can _Promote_ the new version to have new users start using it.
+Using the Zapier CLI:
 
-## Decide on How You Want to Work
+1. Increment the version number in your app’s `package.json` file
+2. Make the necessary updates to the rest of the app’s resources
+3. Run `zapier push` to build and upload the new version to Zapier
```

Comment:
I wonder if it would be helpful to have all of these CLI commands clickable, linking to their docs in the CLI reference, for example https://github.com/zapier/zapier-platform/blob/master/packages/cli/docs/cli.md#push

Reply:
I like that idea! I can make that happen 👍